webapp/app.py

```diff
@@ -172,6 +172,13 @@ def global_template_context():
             component_tabs_file.read(), Loader=yaml.FullLoader
         )
 
+    # Read side-navigation.yaml
+    with open("side-navigation.yaml") as side_navigation_file:
```

Comment:
Could this be loaded once when server starts instead of doing it on every request? I think we load some other static files (version, features) earlier.